workshop/workshop-materials.md

```diff
@@ -15,7 +11,8 @@ PDF versions of the slides we present in this workshop can be found in the [slid
 
 In the `training-modules` repository, each individual module is found in a subfolder of the main page, along with folders containing cheat sheets and setup instructions.
 
-In this training workshop, we will be using the following modules:
+In this training workshop, we will be using the following modules.
```

Comment:
The sentence feels funny ending in a period, but now there is a double "below" (I prefer the [The Lone Bellow](https://www.thelonebellow.com/)). So a bit more tweaking is probably needed. Maybe leave the colon and move the sentence about the links to after the list?
```suggestion
In this training workshop, we will be using the modules listed below.
```

Reply:
Went with this - https://github.com/AlexsLemonade/training-specific-template/pull/144/commits/548a97d2feb480be8560d04f42686ed9f0e900e7

Any feedback lmk!